rhel-edge-84-virt-test.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Test script to test RHEL for edge on RHEL 8.4 virtual.
+# Will prepare the env, download upstream test script and run it.
+
+set -uo pipefail
+source /etc/os-release
+
+function greenprint {
+    echo -e "\033[1;32m${1}\033[0m"
+}
+
+# Initialize repo
+greenprint "Configure yum repository"
```

Comment:
We do not need configure repo here, it's already in jslave vm already.

Reply:
ok. will remove it.